lib/x11regressiontest.pm

```diff
@@ -539,6 +539,7 @@ sub firefox_check_popups {
             # accidentially moving the firefox window around, skip it.
             if (!check_var("DESKTOP", "kde")) {
                 # workaround for bsc#1046005
+                record_soft_failure 'bsc#1046005';
```

Comment:
the bug found on SLE12 SP2 . Does it valid for all versions of SLE including SLE15 ? Does it valid for openSUSE/ Tumbleweed ? Currently developer ask to reproduce issue on openSUSE. If you know that bug is reproducible in all what I mention please add info to bug , otherwise add here version check

Reply:
Also I think you can delete comment about workaround , it's too obvious

Reply:
It's sporadic issue and to do proper fail we need to assert that window is unfocused and then click there again. I've created another MR with bug reference included in file names: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/699 Please review. And I'll close this PR as doesn't make sense.